deeplake/integrations/mmdet/mmdet_.py

```diff
@@ -1216,8 +1219,8 @@ def _train_detector(
 
     if hasattr(model, "CLASSES"):
         warnings.warn("model already has a CLASSES attribute. Will be ignored.")
```

Comment:
What will be ignored? Can we be more specific here?

Reply:
fixed